File: app/src/main/java/com/datarockets/mnchkn/utils/ColorUtil.java
package com.datarockets.mnchkn.utils;

import com.amulyakhare.textdrawable.util.ColorGenerator;

public class ColorUtil {

    public static String generatePlayerAvatarColor() {
        ColorGenerator generator = ColorGenerator.MATERIAL;
        int generatedColor = generator.getRandomColor();
        return "#" + Integer.toHexString(generatedColor).substring(2);
    }

}

File: app/src/main/kotlin/com/datarockets/mnchkn/utils/ColorUtil.kt
package com.datarockets.mnchkn.utils

import com.amulyakhare.textdrawable.util.ColorGenerator

object ColorUtil {

    fun generatePlayerAvatarColor(): String {
        val generator = ColorGenerator.MATERIAL
        val generatedColor = generator.randomColor
        return "#" + Integer.toHexString(generatedColor).substring(2)
    }

}
